sdk/azurestack/Microsoft.Azure.Management.AzureStack/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft AzureStack Management Library in Azure")]
+[assembly: AssemblyDescription("Provides Microsoft AzureStack registration operations in Azure")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Can we mark this as 0.1.0.0, since this is a first release? Anything below 1.0.0.0 is considered preview